src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -84,6 +100,16 @@ public class InventoryWardBrowser extends ModalJFrame implements InventoryListen
     private int[] pColumwidth = { 150, 150, 100, 100, 150 };
     private JComboBox<Object> stateComboBox;
     private JLabel stateLabel;
+    JButton next;
+    JButton previous;
+    JComboBox pagesCombo = new JComboBox();
+    JLabel under = new JLabel("/ 0 Page");
```

Comment:
Shouldn't these fields be private, pagesCombo be instantiated in a getter, JLabel text been defined in a language bundle...